src/ontology/mondo-ingest.Makefile

```diff
@@ -461,7 +461,6 @@ tmp/merged.owl: mondo-ingest.owl tmp/mondo.sssom.ttl tmp/mondo.owl
 $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db $(MAPPINGSDIR)/rejected-mappings.tsv
 	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
 	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical-2.sssom.tsv
-	$(MAKE) pip-bioregistry
```

Comment:
## `pip install --upgrade`
@matentzn I see what you're saying in [this comment](https://github.com/monarch-initiative/mondo-ingest/pull/648#pullrequestreview-2258517161). Normally we do a build in these PRs, but it's possible we may have opted to skip that here if we determined these were all non-functional changes.

What you're saying is that, since `$(MAKE) pip-bioregistry` was added, likely upgraded some of its dependencies, which may have been existing dependencies that we use elsewhere in `mondo-ingest`, and that these upgrades may have different behavior than their non-uprgraded versions. So when removing `$(MAKE) pip-bioregistry`, this could result things behaving differently--possibly differences in outputs, or even errors.

So to be sure, we should run a build. I'll do that now.

Reply:
Build is ready for review:
- #649

Reply:
Awesome, thanks! All good!